### DISCUSSION TITLE
Unable to apply GitHub Student Pack

### DISCUSSION THREAD

**Discussion by saifalisew1508** [community/community#47025] [GitHub Education]
earlier `i had github pro` because i had github student pack but recently it `expired` because i am still a student so i tried to `apply again` for github student but it is having a bug and `i want to use github pro but Unable reapply to github student pack again.`
I used a `new gmail` for this still it is not happening.

`There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.`

![Screenshot_20230211-070811_Chrome](https://user-images.githubusercontent.com/80207551/218232261-f8bad13c-e148-429e-b658-2a37720ce4a0.jpg)

**Comment by jge162** [comment]
Hello @saifalisew1508 You will need to contact support to get help with the alert. It could be because gmail is not a school email (my guess) but, reach out to [support](https://support.github.com/contact/education)

**Reply by samus-aran** [reply]
Thanks @jge162 👍 

This message can be an indication of many things that only the Support team can answer and resolve. But I do not suspect it is the email domain.

So please do contact [Support](https://support.github.com/contact) and they will explain the details of this message.
